dev-util/cproto/cproto-4.7o.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PV="4.7o"
+MY_P=${PN}-${MY_PV}
+S=${WORKDIR}/${MY_P}
+
+DESCRIPTION="Generate C function prototypes from C source code"
+HOMEPAGE="https://invisible-island.net/cproto/"
+SRC_URI="ftp://ftp.invisible-island.net/cproto//${MY_P}.tgz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+
+DEPEND="sys-devel/flex
+	virtual/yacc"
```

Comment:
I assume you checked it installs the same files as `4.7e`?

Reply:
Yes, it does install the same files as in 4.7e